### PR TITLE
contribution question: show "Less than equal share" in red when a negative value is chosen #2825

### LINKED ISSUE #2825: contribution question: show "Less than equal share" in red when a negative value is chosen
This is to minimize students choosing negative values by mistake

### PR DESCRIPTION
**Pull request by rcyq**
Fixes #2825

### CODE REVIEW

**Comment by damithc** [comment]
@LowWeiLin for final review please. 
BTW, do we even need JS for this? I suggested JS because I didn't know that we can colorize options of a dropdown box.

File: src/main/webapp/js/feedbackSubmissionsEdit.js [hunk 1]
```diff
@@ -27,12 +27,57 @@ $(document).ready(function () {
...
+    readyContribQuestions();
+
     readyConstSumQuestions();
     updateConstSumMessages();
 
     readyRubricQuestions();
 });
 
+//Ready contrib questions for answering by user
+function readyContribQuestions() {
```
Comment (damithc) [inline src/main/webapp/js/feedbackSubmissionsEdit.js]: ready is a noun as well as a verb. 'prepare' is less ambiguous.

**Comment by rcyq** [comment]
@LowWeiLin please review

File: src/main/webapp/js/feedbackSubmissionsEdit.js [hunk 2]
```diff
@@ -27,14 +27,42 @@ $(document).ready(function () {
...
 
-    readyRubricQuestions();
+    prepareRubricQuestions();
 });
 
-//Ready rubric questions for answering by user
-function readyRubricQuestions() {
+// Prepare contrib questions for answering by user
+function prepareContribQuestions() {
+    // Set dropdown value to be colored if negative value is select
```
Comment (damithc) [inline src/main/webapp/js/feedbackSubmissionsEdit.js]: Just negative? 
select --> selected

File: src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java [hunk 3]
```diff
@@ -757,19 +757,36 @@ private String getContributionOptionsHtml(int points){
...
     }
     
+    /**
+     * Return the CSS color of different point
+     */
+    private String getContributionOptionsColor(int points){
+        if(points == Const.POINTS_NOT_SURE || points == Const.POINTS_EQUAL_SHARE){
+            // Not sure, Equal Share
+            return "color_neutral";
+        } else if ( points < 100){
```
Comment (LowWeiLin) [inline src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java]: Can use Const.POINTS_EQUAL_SHARE instead of 100 here.

**Comment by LowWeiLin** [comment]
Looks good, only one minor change requested.

**Comment by LowWeiLin** [comment]
Looks good to merge

**Pull request merged**